spec/directConnectionProvider.spec.js

```diff
@@ -1,76 +1,103 @@
 describe('DirectConnectionProvider', function() {
+  var path = require('path');
   var ConnectionProvider = require('../src/connection/directConnectionProvider');
   var logger = require('../src/logger')(3);
 
   var downloadDriversMock = require('./directConnectionProvider/downloadDriversMock')();
   var mockUrl = 'http://localhost';
   var directConnectionProvider;
-  var binaries;
+  var testBinaries;
   beforeAll(function(done) {
     process.env.NO_PROXY = process.env.NO_PROXY || 'localhost';
     downloadDriversMock.start().then(function(port) {
 
       mockUrl += ':' + port;
 
-      binaries = {
+      testBinaries = {
         chromedriver:
           {
             version: '{latest}',
             filename: 'chromedriver',
             executable: 'chromedriver-{latest}',
             baseurl: mockUrl,
-            latestVersionUrl: mockUrl + '/LATEST_RELEASE'
+            latestVersionUrl: mockUrl + '/LATEST_RELEASE',
+            url: mockUrl + '/{latest}/chromedriver_win32.zip',
           },
-          chromedriverGHConfig:
+        chromedriverGHConfig:
           {
             version: '{chrome.latest}',
             latestVersionFileUrl: mockUrl + '/driverVersions.json',
             filename: 'chromedriver',
             executable: 'chromedriver-{chrome.latest}',
             baseurl: mockUrl,
-            latestVersionUrl: mockUrl + '/LATEST_RELEASE_{chrome.latest}'
+            latestVersionUrl: mockUrl + '/LATEST_RELEASE_{chrome.latest}',
+            url: mockUrl + '/{chrome.latest}/chromedriver_win32.zip',
+          },
+        chromedriverLocal:
+          {
+            localPath: path.join(__dirname, "directConnectionProvider/mockChromeDriver.js"),
+            version: '{latest}',
+            filename: 'chromedriver',
+            executable: 'chromedriver-{latest}',
+            baseurl: mockUrl,
```

Comment:
I suggest to leave only the relevant params for this usecase, this is a form of documentation and a test that no other params are used unintentionally.

Reply:
and also a test that no instance params are modified in a wrong way